src/assets/index.js

```diff
@@ -1,3 +1,12 @@
+export { default as icHand } from './icons/ic_hand.svg';
+export { default as icTravel } from './icons/ic_travel.svg';
+export { default as icWish } from './icons/ic_wish.svg';
+export { default as icSearch } from './icons/ic_search.svg';
+export { default as icMassage } from './icons/ic_massage.svg';
```

Comment:
여기 오타가 있구만! 수정 부탁해 ㅎㅎ

```suggestion
export { default as icMessage } from './icons/ic_message.svg';
```